### ISSUE TITLE
** (tauon.py:13): CRITICAL **: 11:16:29.988: gst_alsa_device_new: assertion 'caps' failed

### ISSUE THREAD

**Issue by portal2poe** [Taiko2k/TauonMusicBox#629]
If I'm posting this incorrectly, please correct me!  

I Hadn't have Tauon running for several months, updated my RasberryPi4 to the latest version of Ubuntu (21.10), and Tauon is not happy.   Anyone have any advice?

I've uninstalled/reinstalled through flatpak, and am seeing this error when running from the terminal:

Window size: [915, 1002]
Gtk-Message: 11:16:23.639: Failed to load module "canberra-gtk-module"
GTK theme is: yaru-dark
Detected running as Flatpak
Installed as Flatpak
User files location: /home/nathan/.var/app/com.github.taiko2k.tauonmb/data/TauonMusicBox
Found XDG-Downloads: /home/nathan/Downloads
Found XDG-Music: /home/nathan/Music
Install directory: /app/bin
Database loaded in 1.484 seconds.
Updating database to version 63
Applying scale based on xft setting
Using UI scale: 1.0
Copy tray icons to data directory...
Could not connect to gnome media keys
Start controller server
Init GStreamer...
Using SDL version: 2.0.16

** (tauon.py:13): CRITICAL **: 11:16:29.988: gst_alsa_device_new: assertion 'caps' failed
/app/bin/com.github.taiko2k.tauonmb.sh: line 17:    13 Segmentation fault      (core dumped) python3 /app/bin/tauon.py "$@"

**Comment by Taiko2k** [comment]
Looks like GStreamer is having an issue there. I assume you switched to GStreamer due to issues with Phazor? If you want to switch back to Phazor, you'll need to reset the player by going do the "User files location" (listed above), and deleting `state.p` and `state.p.backup`. (I plan on adding a config setting to make switching this easier next release)

**Comment by portal2poe** [comment]
Just getting back to this! I ended up removing all the /usr settings for Tauon, reinstalling, and now its running...but yes, now when I switch to GStreamer the app crashes. But I'm fine with Phazor, just playing local music.

The whole reason I'm messing around with this is that Mp3  all sound horrible (bad distortion etc) after the upgrade to ubuntu 21.10 on Tauon.  They sound fine on VLC, audacity etc, but Tauon is my favorite player, so I've been messing around trying to get it to work.  I've reinstalled all the ubuntu-restricted-extras, and any other music/video packages i can find "how to"s on...but no luck.  Any suggestions?
